Update create-dapp-app docs (#462)
aptos-labs/developer-docs: apps/nextra/pages/en/build/create-aptos-dapp/templates/digital-asset.mdx

    <FileTree.Folder name="images">
      <FileTree.File name="1.jpeg" />
      <FileTree.File name="2.jpeg" />
      <FileTree.File name="3.jpeg" />
    </FileTree.Folder>
    <FileTree.Folder name="metadata">
      <FileTree.File name="1.json" />
      <FileTree.File name="2.json" />
      <FileTree.File name="3.json" />
    </FileTree.Folder>
  </FileTree.Folder>
</FileTree>

The following steps will break down how to do that.

<Steps>
### Create an `asset` folder for your collection.

### Make a `collection.json` file.
All collections are **required** to have the following fields:
```json filename="collection.json"
{
  "name": "Aptos NFTs",                         
  "description": "My NFT Collection on Aptos.", 
  "image": "",            
  "external_url": "https://your_project_url.io"
}

```
<Callout type="info">
The `image` field of `collection.json` above will be filled in automatically when we upload the images, so we can leave it empty.
</Callout>

### Add an image for the collection itself in the `asset` folder.

### Create a folder called `images` in the `asset` folder.

### Add all potential NFT images to the `images` folder.

> These images must have the same file extension for this template app to work. 

Reviewer: ```suggestion
All images must have the same file extension for this template app to work. 
```

      <FileTree.File name="2.jpeg" />
      <FileTree.File name="3.jpeg" />
    </FileTree.Folder>
    <FileTree.Folder name="metadata">
      <FileTree.File name="1.json" />
      <FileTree.File name="2.json" />
      <FileTree.File name="3.json" />
    </FileTree.Folder>
  </FileTree.Folder>
</FileTree>
</Steps>

## Publish the Move Contract

<Steps>
### Prepare an account, or create one with an Aptos wallet like [Petra](https://petra.app/docs/use#create-a-new-account).

> You will need this to run the contract to create a collection. Make sure to make the account on the same network as your template project.

Reviewer: ```suggestion
You will need this to run the contract to create a collection. Make sure to set up the account on the same network as your template project.
```

### Use the below command to create the contract to mint your collection.

```bash npm2yarn
npm run move:init
```

> The CLI will prompt you to provide the secret for your wallet in order for the tool to make the collection on your behalf.

Reviewer: ```suggestion
The CLI will prompt you to provide the private key for your wallet in order for the tool to make the collection on your behalf.  If you provide no input, it will generate one for you.
```

> This will create the contract that handles minting your collection, as well as setting the `VITE_MODULE_ADDRESS` in the `.env` file to indicate which account will own any collections that get minted.

Reviewer: ```suggestion
This will create the contract for minting your collection, as well as setting the `VITE_MODULE_ADDRESS` in the `.env` file to set which account will own the minted collections.
```
